content/en/_index.md

```diff
@@ -165,6 +208,39 @@ more approachable to beginners:
      }
    }
  {{< /tab >}}
+  {{< tab header="Node.js" >}}
+    // Imports the Google Cloud Data Loss Prevention library
+    const DLP = require('@google-cloud/dlp');
+
+    // Instantiates a reusable client object
+    const dlp = new DLP.DlpServiceClient();
```

Comment:
I realize this use of `DLP`/`dlp` is a fairly common JS practice, but it has always bothered me. I'd be interested in learning more about whether this is a bad habit that gets repeated or something we actively encourage.

Reply:
@ace-n @sofisl this comment is still also open - not sure if you folks have any preferences

Reply:
Hm, I'm not sure I understand the concern. This is how we have it currently in samples: https://github.com/googleapis/nodejs-dlp/blob/bd3d585a773571bc5eb243f34f9891c4b16151d7/samples/categoricalRiskAnalysis.js#L37.

I would prefer going with custom, but happy to hear out the concern further if I'm not understanding it correctly.

Reply:
The idiom of `DLP` and `dlp` as two different variables bothers me. Is this a thing that's often done, or a thing we want people to do? I personally find this a poor UX, so I'm using this moment to briefly question the status quo.

Reply:
A few ideas: would it be better to:
- label these variables as `library`/`client`?
- drop the library import, and directly initialize the client?

Of these, I feel like the first (`library`/`client`) is the easiest to understand.

Reply:
These variables should be labeled as library/client DLP/dlp, respectively.

Reply:
@sofisl to clarify - are you endorsing the following?

```
const library = require(...)

const client = library()
```

Reply:
I'm not asking that we change this, I'm asking if this common Node.js language practice is something we recommend. if so, this is fine. If not, then it seems like we should change it.

Reply:
@grayside, sorry for misunderstanding, yes this is what we currently recommend in our guidelines, insofar as we don't recommend renaming the library that is exported (e.g., `const {thing} = require('import')`, vs `const {thing} as anotherThing = require('import')`

Personally, I think renaming the export is more confusing if someone wanted to go digging into source code. After discussing offline with Ace and understanding the context a bit better, I'd rather us rename the client to something less confusing rather than the library:

```
  // Imports the Dlp library
  const {DlpServiceClient} = require('@google-cloud/dlp').v2;

  // Instantiates a client
  const dlpClientInstance = new DlpServiceClient();
```

Reply:
I'm comfortable with whatever we believe is idiomatic and good UX for Node.js developers, whether that's the existing approach I personally find confusing, or renaming the client.

I understand in python they often use `client` or (to disambiguate) `dlp_client`, so your proposal makes sense to me.